files/acl/pre-wp-utils.php

```diff
@@ -81,6 +81,10 @@ function validate_path( $file_path ) {
 function sanitize_and_split_path( $file_path ) {
 	list( $pre_wpcontent_path, $post_wpcontent_path ) = explode( '/wp-content/uploads/', $file_path, 2 );
 
+	if ( 0 === strpos( $post_wpcontent_path, 'sites/' ) ) {
+		$post_wpcontent_path = preg_replace( '#^sites\/\d+\/#', '', $post_wpcontent_path );
```

Comment:
Slash escaping is redundant here, because the delimiters are `#`.
```suggestion
		$post_wpcontent_path = preg_replace( '#^sites/\d+/#', '', $post_wpcontent_path );
```

Reply:
Good call!